Read only function call - Application error: a client-side exception has occurred

**Describe the bug**
A clear and concise description of what the bug is.
Stacks Explorer throws error on read only function calls. Happens to multiple NFT contracts such as Miami Degens, Megapont, Bitcoin Monkeys and others. Was working a few weeks before.

Transaction ID 
Address 
Block#  
Time stamp

**To Reproduce**
1. Go to [https://explorer.stacks.co/sandbox/contract-call?chain=mainnet](https://explorer.stacks.co/sandbox/contract-call?chain=mainnet)
2. Paste miami-degens contract SP1SCEXE6PMGPAC6B4N5P2MDKX8V4GF9QDE1FNNGJ.miami-degens
3. Click on `Get contract`
4. Scroll down to `get-owner (read_only function)`
5. Write a token id
6. Click on `Call function`
7. See error

**Expected behavior**
It should show the address of the owner for the specific NFT id.

**Screenshots**
console.log:
```ts
instrument.js:109
[debug] pageProps.dehydratedState undefined
-----------------------------------------------------------------------
instrument.js:109
TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)
    at ln (recoil.js:5279:36)
    at recoil.js:5328:41
    at Array.map (<anonymous>)
    at ln (recoil.js:5328:4)
    at recoil.js:5289:32
    at Array.map (<anonymous>)
    at ln (recoil.js:5289:18)
    at recoil.js:5328:41
    at Array.map (<anonymous>)
    at ln (recoil.js:5328:4)
    at dn (recoil.js:5362:10)
    at mi._keyMapper (recoil.js:7105:21)
    at mi.get (recoil.js:7047:31)
    at recoil.js:7234:42
    at fn ([...param]-1b9891ff8f74023c.js:1:121082)
    at ro (react-dom.production.min.js:157:137)
    at Hu (react-dom.production.min.js:267:460)
    at Pi (react-dom.production.min.js:250:347)
    at _i (react-dom.production.min.js:250:278)
    at xi (react-dom.production.min.js:250:138)
    at vi (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Wl (react-dom.production.min.js:122:325)
    at Ql (react-dom.production.min.js:123:61)
    at D (scheduler.production.min.js:16:224)
    at w.port1.onmessage (scheduler.production.min.js:12:346)
-----------------------------------------------------------------------
instrument.js:109
Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=TypeError%3A%20Do%20not%20know%20how%20to%20serialize%20a%20BigInt for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
    at ka (react-dom.production.min.js:140:47)
    at react-dom.production.min.js:150:265
    at Io (react-dom.production.min.js:176:171)
    at Hu (react-dom.production.min.js:271:134)
    at Pi (react-dom.production.min.js:250:347)
    at _i (react-dom.production.min.js:250:278)
    at xi (react-dom.production.min.js:250:138)
    at vi (react-dom.production.min.js:243:163)
    at react-dom.production.min.js:123:115
    at t.unstable_runWithPriority (scheduler.production.min.js:18:343)
    at Wl (react-dom.production.min.js:122:325)
    at Ql (react-dom.production.min.js:123:61)
    at D (scheduler.production.min.js:16:224)
    at w.port1.onmessage (scheduler.production.min.js:12:346)
```
screen shot:
![image](https://user-images.githubusercontent.com/93231870/178794495-5f171a01-5f65-40ff-bbb2-667dfbd9f212.png)



**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome

@andresgalante this would be the related issue per our conversation in Discord.

On my end, testing with [citycoins-tardis-v3](https://explorer.stacks.co/txid/SP2NS7CNBBN3S9J6M4JJHT7WNBETRSBZ9KPVRENBJ.citycoin-tardis-v3?chain=mainnet),  I am unable to call the read-only functions, e.g. `total-supply-mia`

![Screenshot from 2022-08-23 17-37-53](https://user-images.githubusercontent.com/9038904/186289807-15fe4aed-4277-42f5-96aa-a122253c4595.png)

This results in the same message:

![Screenshot from 2022-08-23 17-38-03](https://user-images.githubusercontent.com/9038904/186290010-b927ce99-5713-4cb2-81ec-2929bcf3ca03.png)

Which when inspecting the console reveals:

```
instrument.js:109 TypeError: Do not know how to serialize a BigInt
    at JSON.stringify (<anonymous>)

instrument.js:109 Error: Minified React error #31; visit https://reactjs.org/docs/error-decoder.html?invariant=31&args[]=TypeError%3A%20Do%20not%20know%20how%20to%20serialize%20a%20BigInt for the full message or use the non-minified dev environment for full errors and additional helpful warnings.
```

--- a/src/sandbox/components/screens/call-functions/components/available-functions.tsx
+++ b/src/sandbox/components/screens/call-functions/components/available-functions.tsx
@@ -220,6 +220,12 @@ const FunctionSingleView = () => {
             console.log(e);
           }
         } else {
+          Object.values(final).forEach(value => {
+            if (typeof value?.value === 'bigint') {
+              value.value = value.value.toString();
+            }
+            return value;
+          });
           setReadonly(Object.values(final));
         }
       } catch (e) {

